### ISSUE TITLE
Hard-coded color?

### ISSUE THREAD

**Issue by avonmoll** [simurai/duotone-dark-syntax#26]
With the following custom scheme:
![image](https://cloud.githubusercontent.com/assets/4260850/20679460/01fe4964-b569-11e6-8605-d1303b2fb074.png)

I still have some light-purple colors showing:
![image](https://cloud.githubusercontent.com/assets/4260850/20679486/1c686ff0-b569-11e6-99f8-d05b576d9635.png)

Is this the intended behavior?

**Comment by simurai** [comment]
No, the `conditional...` should also have the Uno hue (orange).

Could you paste that code snippet here? I'll try to reproduce it. And also, what's the grammar (language shown in the bottom right of the status-bar)?

**Comment by avonmoll** [comment]
The grammar is Python. As a slight aside, would it be possible to allow the user to set a custom background color of the theme? It's possible to simply change the stylesheet, but then user must change stylesheet back when switching to other themes. Just a thought!

``` python
def compute_empirical_conditional_distribution(var1_values, var2_values):
    """
    Given two sequences of values (corresponding to samples from two
    random variables), compute the empirical conditional distribution of
    the first variable conditioned on the second variable.

    Inputs
    ------
    - var1_values: list (or 1D NumPy array or some other iterable) of values
        sampled from, say, $X_1$
    - var2_values: list (or 1D NumPy array or some other iterable) of values
        sampled from, say, $X_2$, where it is assumed that the i-th entries of
        `var1_values` and `var2_values` co-occur
    Output
    ------
    - conditional_distributions: a dictionary consisting of dictionaries;
        `conditional_distributions[x_2]` should be the dictionary that
        represents the conditional distribution $X_1$ given $X_2 = x_2$
    """
    conditional_distributions = {x2: {} for x2 in set(var2_values)}

    # -------------------------------------------------------------------------
```

**Comment by simurai** [comment]
Thanks.. I can reproduce this too and should be fixed with the next update.

> As a slight aside, would it be possible to allow the user to set a custom background color of the theme? It's possible to simply change the stylesheet, but then user must change stylesheet back when switching to other themes.

Well, you could limit it to just this theme by adding `.theme-duotone-dark-syntax` in front. Like this:

```less
.theme-duotone-dark-syntax atom-text-editor::shadow,
.theme-duotone-dark-syntax atom-text-editor {
  background-color: darkred;
}
```

Although changing it in your `styles.less` file might make it not match everywhere. For example the active tab will probably be different.

Here a theme where you can change everything: https://github.com/simurai/chameleon-ui + https://github.com/simurai/chameleon-syntax. But both need to be installed to make it work. It's still kinda experimental though.. that's why it's not published. 😄
